tools/pylib/boututils/showdata.py

```diff
@@ -63,17 +63,30 @@ def showdata(vars, titles=[], legendlabels = [], surf = [], polar = [], tslice =
     showdata([var1,var2], surf = [1,0], polar = [0,1])
 
     Movies require FFmpeg to be installed.
+    
+    fps sets frames per second in movie, default: 28 fps
 
+    dpi sets the resolution of the movie, default: 200 dpi
+    
     The tslice variable is used to control the time value that is printed on each
     frame of the animation.  If the input data matches the time values found within
     BOUT++'s dmp data files, then these time values will be used.  Otherwise, an
     integer counter is used.
 
+    The cmap variable (if specified) will set the colormap used in the plot
+    cmap must be a matplotlib colormap instance, or the name of a registered matplotlib colormap 
+
     During animation click once to stop in the current frame. Click again to continue.
 
     global_colors = True: if "vars" is a list the colorlevels are determined from the mximum of the maxima and and the minimum of the  minima in all fields in vars.
 
     symmetric_colors = True: colorlevels are symmetric.
+
+    clear_between_frames: None all plots except line plots will clear between frames
+                          True all plots will clear between frames
+                          False no plots will clear between frames
+
+    return_animation = True matplotlib animation object will be returned.
```

Comment:
I guess there is no reason to not return it ...
If you don't want it, you just ignore the return value ...

Reply:
You might be right. I was trying to keep the default behavior consistent. Previously if the animation was saved to a file the plot wouldn't be shown. I thought it might make sense to only show the plot on screen if it wasn't being otherwise saved or returned.